torch/csrc/utils/disable_torch_function.cpp

```diff
@@ -127,6 +136,43 @@ PyObject* THPModule_disable_torch_function(PyObject *self, PyObject *a) {
   END_HANDLE_TH_ERRORS
 }
 
+PyObject* THPModule_disable_torch_dispatch(PyObject *self, PyObject *a) {
```

Comment:
Not sure why you need an implementation here (and above for torch function)? Shouldn't these never be called?

Reply:
They do get called if you `super().__torch_dispatch__` !

Reply:
Ho ok!
Sound good!